src/e2e/puppeteer-environment.ts

```diff
@@ -29,8 +29,6 @@ const PuppeteerEnvironment: Environment = {
       .catch((err: Error) => {
         // using `console.log` here to avoid errors or logs being swallowed by vitest
         // all of `console.error`, `console.warn` and `console.info` don't show up in the terminal
-        // eslint-disable-next-line no-console
-        console.log('Could not connect to browserless.')
```

Comment:
Is this change relevant to the PR?

Reply:
Yes, for some reason lint didn't like it after some changes. Most likely after I merged with main.

Reply:
We probably want to keep this logging in, as it informs the user if there is any issue with connecting to browserless. The linter does not complain in `main`, so I suspect that this is a regression.